py-functions-amazon/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
So, if I see it correctly, this script is mainly there to compensate for the fact that we have apollo and non-apollo functions in the repo, right? Just FYI: We will be throwing all non-Apollo stuff out very soon.